test/utxo-select.test.js

```diff
@@ -715,8 +718,8 @@ describe('UTXO Select', () => {
     est.amount = 256n;
     deepStrictEqual(t(est, est.biggest, true), {
       amounts: [512n],
-      change: 221n,
-      expFee: 191n,
+      change: 66n,
```

Comment:
change explained by fix on L683

`221 - ( 256 - 100) = 65`

I'm not sure where the 1n fee difference comes from.

Reply:
the difference comes from size of serialization:

`length(varint(1)) != length(varint(2))`